multinode-demo/stop_nodes.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+ip_addr_file=$1
+remote_user=$2
+
+usage() {
+  echo -e "\\tUsage: $0 <IP Address array> <username> [path to ssh keys]\\n"
```

Comment:
How about this style: https://github.com/solana-labs/solana/blob/master/multinode-demo/gce_multinode.sh#L11-L31